Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -15,73 +15,64 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     public static class DispatcherHelper
     {
         /// <summary>
-        /// Executes the given function asynchronously on UI thread of the main view.
+        /// Executes the given function on a given view's UI thread. The default view is the main view.
```

Comment:
>given view [](start = 45, length = 10)

This is more the *main view* UI thread for this one.

Reply:
Thanks! Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3119/commits/d319fb937946904619d1d0842dd69d03d74c6645 😊